src/dotnet/CommonLocalizableStrings.resx

```diff
@@ -520,28 +520,26 @@
   <data name="NoRestoreDescription" xml:space="preserve">
     <value>Does not do an implicit restore when executing the command.</value>
   </data>
-  <data name="CannotBeNullOrWhitespace" xml:space="preserve">
-    <value>Cannot be null or whitespace.</value>
-  </data>
-  <data name="ContainInvalidCharacters" xml:space="preserve">
-    <value>Contains one or more invalid characters: {0}</value>
-  </data>
   <data name="NuGetConfigurationFileDoesNotExist" xml:space="preserve">
     <value>NuGet configuration file {0} does not exist.</value>
   </data>
-  <data name="ToolSettingInvalidRunner" xml:space="preserve">
-    <value>The tool's settings file has non "dotnet" as runner.</value>
+  <data name="ToolSettingsInvalidXml" xml:space="preserve">
+    <value>Invalid XML: {0}</value>
+  </data>
+  <data name="ToolSettingsMissingCommandName" xml:space="preserve">
+    <value>Tool command name is missing.</value>
   </data>
-  <data name="ToolSettingMoreThanOneCommand" xml:space="preserve">
-    <value>The tool's settings file has more than one command defined.</value>
+  <data name="ToolSettingsMissingEntryPoint" xml:space="preserve">
+    <value>Tool entry point is missing.</value>
   </data>
```

Comment:
I would love to find better language than "entry point" but I can't come up with any.

Reply:
The C# compiler uses the following if you attempt to compile an exe without a Main method:
`Program does not contain a static 'Main' method suitable for an entry point`

Reply:
The runtime gives the following when attempting to execute a PE file without an entry point:
`Entry point not found in assembly.`

Reply:
In this context, it is that the tool defines a command with a missing `EntryPoint` (i.e. the assembly to execute) setting.  This would only happen if users published a tool with a malformed settings XML file.

This should really be `Command 'bar' is missing an entry point setting.` since it is for a particular command (currently only one is supporter per tool).

The resulting error would currently be:

```
The settings file in the tool's NuGet package is invalid: Tool entry point is missing.
Tool 'foo' failed to install. Please contact the tool author for assistance.
```

Reply:
I've changed the message to my recent suggestion.